.github/workflows/stale.yml

```diff
@@ -0,0 +1,48 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: "Close Stale Issues"
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  # All other permissions are set to none
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-label: 'stale'
+          exempt-issue-labels: 'not-stale'
```

Comment:
Where I've seen this used before, there was a label for "awaiting reply" so you could make it so issues only go stale if the person that opened the issue stops responding. Is that something we should consider?

Reply:
I think that behavior is largely covered, just in a slightly different way.  The issue will be marked with the `stale` label after the period of inactivity and if anyone (author or otherwise) response, the plugin will remove the `stale` label and the clock resets.

We could change the label to be `awaiting reply` but I think that is less clear in this context than `stale`.  We aren't specifically waiting for a reply and if you were to look at an issue with that label, it's wouldn't be as clear that it is considered stale.

Reply:
Yeah. I agree with Ryan’s assessment. Simply commenting is fine.

And I think personally that “awaiting-response”, unless automated to be added / removed appropriately, could cause a bit more confusion about which issues should be looked at or not.

whereas “stale” is clear that it’s been untouched for basically half a year (presently 180 days). That’s sufficient time to be able to not need to assess it further.

Reply:
And sometimes people submit issues, and GitHub only allows one person as the author / submitter, but it’s clear from the discussion that 2+ people are working on it

eg from a company maybe submitting an issue. In which case a reply from either should be sufficient to make it not stale, but that would be hard to automate.

Reply:
EDITED - Sorry said I agreed with Ryan’s assessment but I meant Dan’s assessment. Updated to reflect.